### PR TITLE
Add search feature #61

### PR DESCRIPTION
**Pull request by Dominik26-1**

### CODE REVIEW

**Review by MikeVarchola — APPROVED** [review]

File: bcitflex/model/enum.py
Comment (jonbiemond) [inline bcitflex/model/enum.py:1]: What do you think about refactoring this into `/app_functions/course_query.py`? All the other classes in `/model` are representations of database objects.

**Review by jonbiemond — CHANGES_REQUESTED** [review]
Hey @Dominik26-1, this looks awesome! Just one small change if that's alright. I can work on resolving any conflicts in the meantime.

File: bcitflex/model/enum.py
Comment (Dominik26-1) [reply bcitflex/model/enum.py:1]: From architecture perspective is it okay, that all classes which call or use this enum will have import dependendency from course_query.py? Do you consider keep it as separated file but change location?

Comment (Dominik26-1) [reply bcitflex/model/enum.py:1]: Done

**Review by Dominik26-1 — COMMENTED** [review]
done refactor

**Review by jonbiemond — APPROVED** [review]

**Comment by Dominik26-1** [comment]
Can you merge my pull request?

**Pull request merged**